t/unit/contrib/test_pytest.py

```diff
@@ -32,3 +34,37 @@ def test_noop():
         result.stdout.fnmatch_lines_random(
             "*PytestUnknownMarkWarning: Unknown pytest.mark.celery*"
         )
+
+
+@pytest.fixture
+def fixed_import(mocker):
+    def import_module(module, package=None):
+        want_reload_module = sys.modules.get(module, None)
+        if want_reload_module:
+            importlib.reload(want_reload_module)
+        return importlib.import_module(module, package=package)
+
+    mocker.patch('celery.loaders.base.BaseLoader.import_module', side_effect=import_module)
+
+
+@pytest.fixture
+def celery_includes():
+    return [
+        't.unit.contrib.proj.foo'
+    ]
+
+
+def test_pytest_import_passes(celery_app, celery_worker):
+    task = celery_app.send_task("t.unit.contrib.proj.foo.bar")
+    task.get()
+
+
+def test_pytest_second_import_fails(celery_app, celery_worker):
+    task = celery_app.send_task("t.unit.contrib.proj.foo.bar")
+    with pytest.raises(Exception):
```

Comment:
Can we get more specific with the exception type/message?

Reply:
Replaced with `KeyError`